fetchcode/__init__.py

```diff
@@ -0,0 +1,105 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at:
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from ftplib import FTP
+from mimetypes import MimeTypes
+import os
+import tempfile
+from urllib.parse import urlparse
+
+import requests
+
+
+class Response:
+    def __init__(self, location, content_type, size, url):
+        """
+        Represent the response from fetching a URL with:
+        - `location`: the absolute location of the files that was fetched
+        - `content_type`: content type of the file
+        - `size`: size of the retrieved content in bytes
+        - `url`: fetched URL
+        """
+        self.url = url
+        self.size = size
+        self.content_type = content_type
+        self.location = location
+
+
+def fetch_via_http(url, location):
```

Comment:
Lets remove the `via` string from all these methods.

Reply:
Okay got this !!